pkg/agent/config/node_config.go

```diff
@@ -122,6 +128,8 @@ type NodeConfig struct {
 	NodeTransportIPv4Addr *net.IPNet
 	// The IPv6 address on the Node's transport interface. It is used for tunneling or routing the Pod traffic across Nodes.
 	NodeTransportIPv6Addr *net.IPNet
+	// The name on the Node's transport interface. It is used for tunneling or routing the Pod traffic across Nodes.
+	NodeTransportName string
```

Comment:
Why we need another one here when we already have TransportInterface in NetworkConfig?

Reply:
@wenyingd Not sure I was doing this right. Reset `networkConfig.TransportInterface` here https://github.com/antrea-io/antrea/pull/2835/files#diff-18bf746206c8ac217eb16ffb0cf91a6e676e05a7b517dc70aef0260d0871028fR779

Reply:
removed.